assets/src/editor/events/initialization/index.js

```diff
@@ -0,0 +1,25 @@
+export {
+	useEditorInitialized,
+	Consumer,
+} from './context';
```

Comment:
These two exports allow for add-ons, or other distinct component trees in editor interfaces to utilize the "initialized" context.  As long as the component using these is within a `Provider`, they can be utilized.  The provider is exposed via `withEditorInitialization` hoc.  So typical usage would go something like this for `useEditorInitialized` (the hook method).

```jsx
const MyComponent = ( props ) => {
	const initialized = useEditorInitialized();

	return initialized ? <Loading { ...props } /> : <NonLoading { ...props }/>;
};

export default withEditorInitialization( MyComponent );
```

Or this way using the Consumer:

```jsx
const MyComponent = ( { initialized, ...props } ) => {
	return initialized ? <Loading { ...props } /> : <NonLoading { ...props }/>;
};

const App = ( props ) => {
	return <Consumer>
		{ ( initialized ) => (
			<MyComponent initialized={ initialized } { ...props } />
		) }
	</Consumer>
};

export default withEditorInitialization( App );
```

Reply:
I don't think there's any harm in having both here.  The maintenance overhead is virtually nil as `Consumer` is something exposed via the `React.Context` api.  For our _own_ use, we'll likely use the hook more because it's an easier api to work with (imo), but there's no harm in exposing `Consumer` for third-party devs that want to use it because they are familiar with it.